.gitmodules

```diff
@@ -0,0 +1,32 @@
+[submodule "sc-flexy-datapoint-lib"]
+	path = sc-flexy-datapoint-lib
+	url = https://github.com/hms-networks/sc-flexy-datapoint-lib.git
+	branch = master
+[submodule "flexy-azure-connector-generic"]
```

Comment:
why is there a submodule reference to the generic azure connector?

Reply:
Interesting, I never noticed that. I must of deleted it after accidentally grabbing the wrong url without removing it from the .gitmodules

I'll remove that.